src/configs/esbuild.es6.config.js

```diff
@@ -26,6 +26,13 @@ const babelPresetEnv = require('@babel/preset-env')
 const path = require('path')
 const babelPluginClassProperties = require('@babel/plugin-proposal-class-properties')
 const babelPluginInlineJsonImport = require('babel-plugin-inline-json-import')
+const deepMerge = require('deepmerge')
+
+let customConfig
+
+if (process.env.LNG_CUSTOM_ESBUILD === 'true') {
+  customConfig = require(path.join(process.cwd(), 'esbuild.es6.config'))
```

Comment:
What about the _env_?

Reply:
Hey @ricardo-a-alves-alb ,
The ones that are mentioned in the env will be ignored here and the custom config takes the priority here.

Reply:
I've misread the configuration description. By `env`, I thought it meant like `development`, `staging` and `production`. But now I see that it's about `es5` and `es6`.